src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -170,24 +174,28 @@ private void resetOdometry(Pose2d pose) {
 
     // Drive function - slew rate limited to prevent shearing of wheels
     public void drive(double forwardMetersPerSecond, double sidewaysMetersPerSecond, double radiansPerSecond, boolean fieldRelative, boolean rateLimit) {
```

Comment:
Above all functions please add a javadoc entry so in the future we can generate readable documentation for out code. 
[How to write javadoc](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html)